modules/CmpdReg/src/server/routes/CmpdRegRoutes.coffee

```diff
@@ -335,14 +332,38 @@ exports.getLotDependenciesInternal = (lot, user, allowedProjects, includeLinkedL
 			# This way it'know there are experiments linked that aren't readable
 			if !acls.getRead()
 				console.log "Experiment #{experiment.experiment.codeName} is not readable by user #{user.username}"
-				# If the experiiment is not readable we just want to include the acls but not the experiment code table
+				# If the experiment is not readable we just want to include the acls but not the experiment code table
 				# We include the acls so that the user can see that there is an experiment linked that they can't read
 				dependencies.linkedExperiments[idx] = {acls: acls, code: null, name: null, ignored: false}
 			else
-				# The experiment is readable so includ the experiment and acls
-				dependencies.linkedExperiments[idx].acls = acls
+				# The experiment is readable so include the experiment and acls
+				linkedExperiment = dependencies.linkedExperiments[idx]
+				linkedExperiment.acls = acls
+		
+		# Add protocol and analysis group codes to user accessible experiments
+		for linkedExperiment in dependencies.linkedExperiments
+			if not linkedExperiment.acls.getRead()
+				continue
+			experiment = _.findWhere(experiments, {experimentCodeName: linkedExperiment.code})
+			linkedExperiment.protocolCode = experiment.experiment.protocol.codeName
```

Comment:
I think the users may also want to see the protocol name which is buried in the protocol.lsLabels.  Can we pul that up? Perhaps the output should be:

```
"linkedExperiments": [
        {
            "code": "EXPT-00000001",
            "comments": "CMPD-0000001-001",
            "description": "60 results and 280 raw results",
            "ignored": false,
            "name": "4 parameter D-R - 2018-05-08",
            "acls": {
                "read": true,
                "write": true,
                "delete": true
            },
            "protocol": {
                  "code": "PROT-00000001",
                  "name": "BLAHBLAH Protocol"
 ```

Reply:
Done.